### ISSUE TITLE
Traffic Monitor Incomparable stat type int on back-up fields

### ISSUE THREAD

**Issue by shamrickus** [apache/trafficcontrol#4870]
## I'm submitting a ...

- [x] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [x] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:

Currently if TM is passed specific `system` stats, the TM logs will be polluted with the following error:
```ERROR: stat.go:292: 2020-07-09T13:03:34.31486976Z: Adding result from cache-server-1: some stats could not be added: cannot add stat system.astatsLoad: incomparable stat type int; cannot add stat system.lastReloadRequest: incomparable stat type int; cannot add stat system.lastReload: incomparable stat type int; cannot add stat system.inf.speed: incomparable stat type int; cannot add stat system.configReloads: incomparable stat type int; cannot add stat system.configReloadRequests: incomparable stat type int```

## Expected / new behavior:
No errors when `system` stats are present.

## Minimal reproduction of the problem with instructions:
Have you cache server return one of these stats:
```
system.astatsLoad
system.configReloadRequest
system.configReloads
system.inf.name
system.inf.speed
system.lastReload
system.lastReloadRequest
system.notAvailable
system.proc.loadavg
system.proc.net.dev
```

**Comment by ocket8888** [comment]
This issue was fixed two weeks ago: #4825

**Comment by mitchell852** [comment]
> This issue was fixed two weeks ago: #4825

so can this issue be closed?

**Comment by shamrickus** [comment]
I don't think you can reproduce the issue #4871 fixes given that #4825 already fixed the issue.
